opentreemap/importer/templates/importer/partials/species_resolve_popover.html

```diff
@@ -16,6 +13,8 @@
 <input class="popover-correction species-resolver-typeahead"
        data-typeahead-url="{% url 'species_list_view' instance_url_name=request.instance.url_name %}"
        value="{{ field.custom_resolver.suggestion.display_name }}"/>
+<p style="font-size:1.1rem;line-height:1.4rem;">Click "Save" to accept the suggested species or click in the text box
```

Comment:
Personally I think one-off styling like this is easier to understand in the HTML than in some CSS file.